pandas/io/tests/test_date_converters.py

```diff
@@ -138,6 +138,19 @@ def date_parser(date, time):
                                names=['datetime', 'prn']))
         assert_frame_equal(df, df_correct)
 
+    def test_parse_date_column_with_empty_string(self):
+        # GH 6428
+        data = """case,opdate
+                  7,10/18/2006
+                  7,10/18/2008
```

Comment:
yep, I agree that we should not add kwargs to read_csv for something like this. If you want the date parser, your dates should just be able to be parsed with the default settings, otherwise you can just use `to_datetime`. PR for doc clarification certainly welcome!